components/Chat/ChatDetailBody.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle, TextStyle } from 'react-native';
+import { AppTheme } from '../../config/DefaultConfig';
+import useTheme from "../../hooks/useTheme";
+import ThemedText from '../UI/ThemedText';
+import TimeDuration from './TimeDuration';
+
+interface Props {
+    message: string;
+    timeStamp: Date;
+    messageDirect: boolean;
+};
+
+const ChatDetailBody: React.FunctionComponent<Props> = ({
+    message,
+    timeStamp,
+    messageDirect,
+}: Props) => {
+    const theme: AppTheme = useTheme();
+
+    return (
+        <View style={[style.container, messageDirect ? {flexDirection: 'row'}: {flexDirection: 'row-reverse'}]}>
```

Comment:
This can be simplified more like this

const rowStyle = {
    flexDirection: messageDirect ? 'row' : 'row-reverse'
}

and then pass `rowStyle` to style

Reply:
I have fixed this